src/DynamoCoreWpf/Interfaces/IBrandingResourceProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Windows.Media;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    public enum ResourceName
+    {
+        StartPageLogo,
+        AboutBoxLogo
+    }
+
+    public interface IBrandingResourceProvider
+    {
+        ImageSource GetImageSource(ResourceName resourceName);
+        string GetStringResource(ResourceName resourceName);
```

Comment:
I meant for that to be `GetImageSource` and `GetString`, both of which named after the object types they return. I don't really mind `GetImage` and `GetString` too, as long as we don't include `Resource` in the method names (because that's what the interface is already describing).